plone/app/portlets/portlets/navigation.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from Acquisition import aq_inner, aq_base, aq_parent
 from ComputedAttribute import ComputedAttribute
+from plone import api
```

Comment:
After reading this comment from @jensens https://github.com/plone/plone.portlet.collection/pull/15/files#r110555223 I guess it is ok.